vita3k/gui/src/game_selector.cpp

```diff
@@ -92,7 +133,7 @@ void draw_game_selector(GuiState &gui, HostState &host) {
         gui.delete_game_icon = false;
     }
 
-    const float icon_size = static_cast<float>(host.cfg.icon_size);
+    const float icon_size = float(host.cfg.icon_size);
```

Comment:
functional style cast is a bit strange, I think I would keep the static_cast just in case